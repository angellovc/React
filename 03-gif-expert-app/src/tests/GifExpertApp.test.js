import {shallow} from 'enzyme';
import '@testing-library/jest-dom';
import GifExpertApp from '../GifExpertApp';


describe('<GifExpectApp /> Tests', () => {

    test('Snapshot should match', () => {
        const wrapper = shallow(<GifExpertApp />);
        expect(wrapper).toMatchSnapshot();
    });
    
});
