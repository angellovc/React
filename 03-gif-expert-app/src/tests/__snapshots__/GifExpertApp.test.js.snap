// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GifExpectApp /> Tests Snapshot should match 1`] = `
<Fragment>
  <h2>
    Gif Expert App
  </h2>
  <AddCategory
    setCategories={[Function]}
  />
  <hr />
  <ul>
    <GifGrid
      category=""
      key=""
    />
  </ul>
</Fragment>
`;
